src/test/java/seedu/recipe/logic/commands/ExitCommandTest.java

```diff
@@ -9,12 +10,17 @@
 import seedu.recipe.model.ModelManager;
 
 public class ExitCommandTest {
-    private Model model = new ModelManager();
-    private Model expectedModel = new ModelManager();
+    private final Model model = new ModelManager();
+    private final Model expectedModel = new ModelManager();
 
     @Test
     public void execute_exit_success() {
```

Comment:
may want to describe test scenario (valid model): `execute_validModel_success`

Reply:
fixed